packages/js/components/src/table/README.md

```diff
@@ -241,6 +241,7 @@ Name | Type | Default | Description
 `rows` | Array | `null` | (required) An array of arrays of display/value object pairs
 `rowHeader` | One of type: number, bool | `0` | Which column should be the row header, defaults to the first item (`0`) (but could be set to `1`, if the first col is checkboxes, for example). Set to false to disable row headers
 `rowKey` | Function(row, index): string | `null` | Function used to get the row key.
+`noDataLabel` | String | `undefined` | Customize the label to show when there are no rows in the table.
```

Comment:
Nitpicky, we already have a similar prop called [`emptyMessage`](https://github.com/woocommerce/woocommerce/blob/7fad3ca7114ecc57a9972c2470bf45e860787e5c/packages/js/components/src/chart/index.js#L516) in another component. Do you think the same label can be used without too much context difference? It's good with me either way

Reply:
Sure, makes sense I added the change in 

https://github.com/woocommerce/woocommerce/pull/36124/commits/1930f9b95bd55b000cde4a846553b7056ec3b123